src/tests/postgres/compose.yaml

```
@@ -2,7 +2,12 @@ services:
   x:
     image: postgres
     x-defang-postgres:
-
+      maintenance-window: Mon:00:00-Mon:01:00
+      retention:
+        backup-window: 23:30-00:30
+        retention-period: 30
```

make this a Golang duration string (like other durations in compose)
```suggestion
        retention-period: 30s
```